app/components/pagination-nav.js

```diff
@@ -1,4 +1,11 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+  actions: {
+    saveTags() {
+      window.console.log("In pagination-nav saveTags()");
+      this.sendAction('saveTags'); // Trigger the route's route-action
+      return true; // Bubble saveTags action up to hit index/chapters route
```

Comment:
The comment is incorrect. The sendAction() method above it is what's sending it to the route-action.